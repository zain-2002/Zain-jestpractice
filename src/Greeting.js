import React from 'react';
import { View, Text } from 'react-native';

const Greeting = ({ name }) => (
  <View>
    <Text>Hello, {name}!</Text>
  </View>
);

export default Greeting;
