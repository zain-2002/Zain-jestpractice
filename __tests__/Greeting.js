import React from 'react';
import renderer from 'react-test-renderer'; 
import Greeting from '../src/Greeting';

describe('Greeting component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Greeting name="John" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
