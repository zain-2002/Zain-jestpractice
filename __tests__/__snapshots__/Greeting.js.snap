// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting component renders correctly 1`] = `
<View>
  <Text>
    Hello, 
    John
    !
  </Text>
</View>
`;
